_data/simple-icons.json

```diff
@@ -1927,6 +1927,11 @@
             "hex": "000000",
             "source": "https://en.wikipedia.org/wiki/Contactless_payment"
         },
+        {
+            "title": "containerd",
+            "hex": "FFFFFF",
```

Comment:
Based on [the source](https://cncf-branding.netlify.app/projects/containerd/) I'd say this should be `#575757` :thinking: 

```suggestion
            "hex": "575757",
```

Reply:
yeah, I'm also a little confused. As the primary is 575757, however, it's not consistent with what the icon should look like. so can you give me some advice?
official: 
![Screen Shot 2021-08-04 at 10 31 49 AM](https://user-images.githubusercontent.com/18364341/128112536-0e4f72b5-6cf9-4ede-8a0e-4d469d6c5368.png)

set to 575757:
![Screen Shot 2021-08-04 at 10 29 45 AM](https://user-images.githubusercontent.com/18364341/128112380-b78b224a-053b-4f32-8b62-4a5af6708296.png)

Reply:
We aim for the `"hex"` value to be the brand color rather than "the icon" color, hence I think `#575757` makes more sense.